gen/libpng/prologue.jl

```diff
@@ -0,0 +1,7 @@
+## TODO: pending https://github.com/JuliaLang/julia/issues/29420
+
+const PNGCAPI = nothing
+const PNG_BYTES_TO_CHECK = 8
+const tm = Libc.TmStruct
+const time_t = UInt
+const jmp_buf = Nothing
```

Comment:
If `jmp_buf` is only used as an opaque pointer in the library, this is OK. Otherwise, it should be carefully defined for different platforms.